app/controllers/api_controller/manager.rb

```diff
@@ -135,7 +135,7 @@ def update_multiple_collections(is_subcollection, target, type, resources)
         next if r.blank?
 
         rid = parse_id(r, type)
-        if rid && %w(create add).include?(action)
```

Comment:
a couple of collections still support add (from v1.0 days) which newer collections use create (which allows the POST on collection for creation). I'd rather not change that now.

Let's pick a different action name for adding a service_request to a service_order.